docs/specifications/tests/Consistency-TP/consistency03.md

```diff
@@ -6,25 +6,25 @@
 
 ## Objective
 
-All SOA record timers fields must be consistent
+All SOA record timer fields must be consistent
 across all authoritative name servers.
 
-The inconsistency in these different fields for the designated zone
-might result in operational inconsistencies.
+An inconsistency in these fields might result in operational inconsistencies 
+for the designated zone.
 
 There are other test cases that provide consistency tests for other
-SOA fields.
+SOA fields. **(Suggest either mentioning which test cases this refers to, or removing this statement.)**
```

Comment:
I suggest removing this statement.

Reply:
That statement was a comment from @JohanIIS to me and has already been removed through my additional updates some 10 minutes after I merged @JohanIIS 's commit into my branch.